--- gtk+-2.24.9/gdk/Makefile.am.linking	2012-01-25 09:14:51.478513064 +0100
+++ gtk+-2.24.9/gdk/Makefile.am	2012-01-25 10:21:00.522099933 +0100
@@ -201,7 +201,7 @@ introspection_files = 		\
 Gdk-2.0.gir: $(gdktargetlib) Makefile
 Gdk_2_0_gir_SCANNERFLAGS = --warn-all
 Gdk_2_0_gir_INCLUDES = Gio-2.0 GdkPixbuf-2.0 Pango-1.0 cairo-1.0
-Gdk_2_0_gir_LIBS = $(gdktargetlib)
+Gdk_2_0_gir_LIBS = $(gdktargetlib) gmodule-2.0
 Gdk_2_0_gir_FILES = $(introspection_files)
 Gdk_2_0_gir_CFLAGS = $(INCLUDES)
 INTROSPECTION_GIRS += Gdk-2.0.gir
@@ -242,7 +242,7 @@ x11_introspection_files = 		\
 GdkX11-2.0.gir: $(gdktargetlib) Gdk-2.0.gir Makefile
 GdkX11_2_0_gir_SCANNERFLAGS = --warn-all --strip-prefix=Gdk
 GdkX11_2_0_gir_INCLUDES = Gio-2.0 Gdk-2.0 GdkPixbuf-2.0 Pango-1.0 xlib-2.0
-GdkX11_2_0_gir_LIBS = $(gdktargetlib)
+GdkX11_2_0_gir_LIBS = $(gdktargetlib) gmodule-2.0
 GdkX11_2_0_gir_FILES = $(x11_introspection_files)
 GdkX11_2_0_gir_CFLAGS = $(INCLUDES) -L$(top_builddir)/gdk
 INTROSPECTION_GIRS += GdkX11-2.0.gir
--- gtk+-2.24.9/gtk/Makefile.am.linking	2012-01-25 10:32:24.192935729 +0100
+++ gtk+-2.24.9/gtk/Makefile.am	2012-01-25 10:55:39.123799065 +0100
@@ -120,7 +120,7 @@ endif
 libgtkincludedir = $(includedir)/gtk-2.0/gtk
 libadd =								\
 	$(top_builddir)/gdk/$(gdktargetlib)				\
-	$(GTK_DEP_LIBS)
+	$(GTK_DEP_LIBS)  -lgmodule-2.0
 deps =
 
 # libtool stuff: set version and export symbols for resolving
@@ -1000,7 +1000,7 @@ TEST_DEPS = $(DEPS) gtk.immodules
 LDADDS =								\
 	$(gtktargetlib)							\
 	$(top_builddir)/gdk/$(gdktargetlib)                             \
-	$(GTK_DEP_LIBS)
+	$(GTK_DEP_LIBS)  -lgmodule-2.0
 
 if HAVE_INTROSPECTION
 introspection_files = \
@@ -1020,7 +1020,7 @@ Gtk_2_0_gir_CFLAGS = \
 		-UGDK_DISABLE_DEPRECATED \
 		-UGTK_DISABLE_DEPRECATED \
 		-DGTK_TEXT_USE_INTERNAL_UNSUPPORTED_API
-Gtk_2_0_gir_LIBS = $(gtktargetlib)
+Gtk_2_0_gir_LIBS = $(gtktargetlib) gmodule-2.0
 Gtk_2_0_gir_FILES = $(introspection_files)
 INTROSPECTION_GIRS += Gtk-2.0.gir
 
